types/eslint/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/eslint",
-    "version": "8.56.9999",
+    "version": "9.6.9999",
```

Comment:
[Question] Is there any reason to keep this `.9999` versioning?

Reply:
That is what the documentation suggests:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b83cf9ef8b044e69f05b2a00aa7c6cb767a9acd2/README.md?plain=1#L257-L262

The meaning of `.9999` is explained further down in the same README.md file:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b83cf9ef8b044e69f05b2a00aa7c6cb767a9acd2/README.md?plain=1#L694

> The `.9999` is to ensure that local `@types` packages are always newest during local development.

Reply:
Ah, sorry, I'd forgotten about this.